recipes/libltc/all/conanfile.py

```diff
@@ -0,0 +1,71 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+import os
+
+
+class LibltcConan(ConanFile):
+    name = "libltc"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://x42.github.io/libltc/"
+    description = "Linear/Logitudinal Time Code (LTC) Library"
+    topics = ("timecode", "smpte", "ltc")
+    license = "LGPL-3.0"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    _autotools = None
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+        args = []
+        if self.options.shared:
+            args.extend(["--disable-static", "--enable-shared"])
+        else:
+            args.extend(["--disable-shared", "--enable-static"])
+        self._autotools.configure(args=args)
+        return self._autotools
+
+    def build(self):
+        with tools.chdir(self._source_subfolder):
+            autotools = self._configure_autotools()
+            autotools.make()
+
+    def package(self):
+        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
+        with tools.chdir(self._source_subfolder):
+            autotools = self._configure_autotools()
+            autotools.install()
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        os.unlink(os.path.join(self.package_folder, "lib", "libltc.la"))
+
+    def package_info(self):
```

Comment:
```suggestion
    def package_info(self):
        self.cpp_info.names["pkg_config"] = "ltc"
```

https://github.com/x42/libltc/blob/v1.3.1/ltc.pc.in